packages/core/src/tracing/hubextensions.ts

```diff
@@ -225,9 +217,9 @@ export function startIdleTransaction(
 }
 
 /**
- * @private
+ * Adds tracing extensions to the global hub.
  */
-export function _addTracingExtensions(): void {
+export function addTracingExtensions(): void {
```

Comment:
l: Can we mark this as deprecated out of caution. I think we wanna move to dependency injection rather than functions with side-effects.

Reply:
Will do this in a follow up PR since we should make the deprecations the final step after we've moved everything over.